pythonforandroid/bootstraps/service_only/build/src/org/kivy/android/PythonService.java

```diff
@@ -136,7 +131,6 @@ public void onDestroy() {
     @Override
     public void run() {
         PythonUtil.loadLibraries(getFilesDir());
-        mService = this;
```

Comment:
Why remove the mService stuff? Is this because it no longer makes sense when multiple services can run at once? (Honest question, I haven't used the services).

Looks good otherwise, thanks!